tests/execute/restraint/report_result/data/smoke.fmf

```diff
@@ -0,0 +1,10 @@
+summary: Generate a report result file to ensure TMT can read the result.
+description:
+    Test to generate a rstrnt-report-result file to check
+    whether tmt can read the result.
+
+/good:
+    test: rstrnt-report-result --server http://test-example.com report PASS
```

Comment:
Why have you removed the check for the failing test? I think it would be good to verify that the mapping is working as expected.

Reply:
Why don't you expect tmt run to exit '1'? rlRun allows you to check for specific exit code (see other rlRun with "1" as 2nd parameter)

Reply:
I'd say @lukaszachy's proposal for `rlRun "tmt run..." 1` was a bit better way as we don't have anything like `expected warn` defined for the `result` key. The individual `rlAssertGrep` checks should ensure that each of the tests ended as expected.